rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -2143,30 +2146,20 @@ bool spu_thread::do_putllc(const spu_mfc_cmd& args)
 			{
 			case UINT32_MAX:
 			{
-				const auto render = rsx::get_rsx_if_needs_res_pause(addr);
-
-				if (render) render->pause();
+				cpu_thread::suspend_all cpu_lock(this);
```

Comment:
There are many rebase errors here like the removed cmp_rdata check aside from the fact that cpu_thread::suspend_all is a function. also in do_cell_atomic_128_store.

Reply:
Git rebase fucked up the code in this file quite badly, I ended up manually rewriting the changes in it.